components/datatypes/data/order.schema.json

```diff
@@ -61,6 +61,11 @@
           "meta:titleId": "order##xdm:taxAmount##title##81954",
           "meta:descriptionId": "order##xdm:taxAmount##description##51205"
         },
+        "xdm:DiscountAmount": {
```

Comment:
One last thing, we tend to keep property names start with a small case "discountAmount"

Reply:
Oops, yeah, that was a typo